lib/router/src/router.tsx

```diff
@@ -1,14 +1,7 @@
 import global from 'global';
 import React, { ReactNode, useCallback } from 'react';
 
-import {
-  Link,
-  BrowserRouter,
-  useNavigate,
-  useLocation,
-  NavigateOptions,
-  Router,
-} from 'react-router-dom';
+import * as R from 'react-router-dom';
```

Comment:
Renaming exports is just not really supported by the dts-bundle-generator

As discussed @tmeasday we could remove the usage, and use tsc to simply create the deeply nested tree before handing it over to the dts-localizer.

I'm a bit devided on the issue, it's not a great limitation to have, but I really like the output.